doc/050_restore.rst

```diff
@@ -19,46 +19,50 @@ Restoring from a snapshot
 
 Restoring a snapshot is as easy as it sounds, just use the following
 command to restore the contents of the latest snapshot to
-``/tmp/restore-work``:
+``/tmp/restore``:
 
 .. code-block:: console
 
-    $ restic -r /srv/restic-repo restore 79766175 --target /tmp/restore-work
+    $ restic -r /srv/restic-repo restore 79766175 --target /tmp/restore
     enter password for repository:
-    restoring <Snapshot of [/home/user/work] at 2015-05-08 21:40:19.884408621 +0200 CEST> to /tmp/restore-work
+    restoring <Snapshot of [/home/user/work] at 2015-05-08 21:40:19.884408621 +0200 CEST> to /tmp/restore
 
```

Comment:
- I think this would be a good place to add an explanation of how the paths in the snapshot populates the target directory of the restore (see comments below this comment for reasoning – I'm basically suggesting moving and elaborating on your addition further down).
- I'm suggesting the following text that hopefully explains this once and for all:

~~~
You will notice that the paths listed in the snapshot you are restoring will
be the paths populating the target directory of the restore. For example, if
``restic snapshots`` lists the path ``/home/art`` in the snapshot to restore
and you specify ``--target /tmp/restore``, the result will be that ``/home/art``
from the snapshot will be restored as the path ``/tmp/restore/home/art``.
~~~